app/assets/javascripts/notebook_actions.js.erb

```diff
@@ -569,7 +570,12 @@ $(document).ready(function(){
     });
     return false;
   });
+  // Disable/Enable hack for submit button. Remove once over to Rails 5, is default behavior in 5.
+  if ($('#deprecateNotebookReasoning').value == null) {
+    $('#deprecateNotebookSubmit').attr('disabled', true);
+  }
   $('#deprecateNotebookReasoning').keyup(function(){
+    $('#deprecateNotebookSubmit').attr('disabled', false);
```

Comment:
This does not verify that there is actual content in the box, just that a key was pressed and released.  Press an arrow key and release, and it still activates the deprecate button.